app/workers/metrics/record_daily_notifications_worker.rb

```diff
@@ -0,0 +1,34 @@
+module Metrics
+  class RecordDailyNotificationsWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :low_priority, retry: 10
+
+    EVENT_TITLES = %w[
+      welcome_notification_set_up_profile
+      welcome_notification_welcome_thread
+      welcome_notification_customize_feed
+      welcome_notification_twitter_connect
+      welcome_notification_github_connect
+      welcome_notification_customize_experience
+      welcome_notification_discuss_and_ask
+      welcome_notification_download_app
+      welcome_notification_ask_question
+      welcome_notification_start_discussion
+    ].freeze
+
+    def perform
+      # Welcome Notification click events created in the past day, logged by title.
+      EVENT_TITLES.each do |title|
+        event = Ahoy::Event.where(name: "Clicked Welcome Notification").
+          where("time > ?", 1.day.ago).
+          where("properties->>'title' = ?", title)
```

Comment:
🔥

Reply:
@rhymes i was SO AMAZED when i figured out that this is a thing and that i could use it!!!

Reply:
@vaidehijoshi into the the fantasy land of PostgreSQL JSONB :P 

If that ever becomes slow, know that you can add indexes on JSON `properties.title` or any other JSON field